2023/06/14-london/README.md

```diff
@@ -86,11 +86,15 @@ Link: TBD
 
 ### Day 2
 
-- <= 30m: Hellos, Google Doc, Call for Scribes
+- <= 15m: Hellos, Google Doc, Call for Scribes
 - >= 45m: [#125 Malicious Partitioning Without Sorting](https://github.com/patcg/meetings/issues/125)
 - => 25m: [#128 Short presentation on label DP results](https://github.com/patcg/meetings/issues/128)
+- => 1h: [#115 Cross device attribution options](https://github.com/patcg/meetings/issues/115)
 - == 1h: Lunch
-- [Continue Baseline Requirements Review](https://github.com/patcg/meetings/issues/91)
+- == 30m: [Methods for measuring privacy risk](https://github.com/patcg/meetings/issues/129)
```

Comment:
@AramZS , okay to put https://github.com/patcg/meetings/issues/127 after the higher level discussion of methods for measuring privacy risk?

Reply:
Makes sense. Added now.